targets/TARGET_STM/us_ticker_32b.c

```diff
@@ -28,18 +28,16 @@
 #include <stddef.h>
 #include "us_ticker_api.h"
 #include "PeripheralNames.h"
+#include "hal_tick.h"
 
-#if defined(TIM5_BASE)
-  #define TIM_MST TIM5
-#else
-  #define TIM_MST TIM2
-#endif
+// A 32-bit timer is used
+#if !TIM_MST_16BIT
+
+TIM_HandleTypeDef TimMasterHandle;
 
-static TIM_HandleTypeDef TimMasterHandle;
 static int us_ticker_inited = 0;
 
-void us_ticker_init(void)
-{
+void us_ticker_init(void) {
```

Comment:
why did you change the style ? I understand there might be some inconsistency that should be fixed . It was good as it was.